managers/tests/test_unit_model.py

```diff
@@ -0,0 +1,48 @@
+from django.test import TestCase
```

Comment:
these imports are not grouped.
imports from django should be adjacent to each other.
same for managers.
And keep alphabetical order.
https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Done